config/clickhouse/config.xml

```diff
@@ -1,3 +1,16 @@
 <yandex>
     <max_server_memory_usage_to_ram_ratio from_env="MAX_MEMORY_USAGE_RATIO" />
+
+    <zookeeper>
```

Comment:
By default we run sentry without Kafka and zookeeper. If you provide a zookeeper config to clickhouse but zookeeper is not running does this work? 
You can test it by turning off Kafka eventstream (`SENTRY_EVENTSTREAM` sentry setting) and restarting sentry devservices.

Reply:
It seems you would also need to change the SENTRY_USE_RELAY value to false to test this as this seems to cause Kafka and Zookeeper to be started by default. https://github.com/getsentry/sentry/blob/9f4199142f225d0fce4492bbc8e084ab15473eaf/src/sentry/conf/server.py#L1637

Reply:
Locally, it seems to work without Kafka on. But Zookeeper must be running. I think that Zookeeper is not running by default on GitHub, though. Is there a way to make that start up on GitHub automatically?

Reply:
Setting SENTRY_USE_RELAY to false does not allow me to start up Zookeeper locally ("Service `zookeeper` is not known or not enabled.)

Reply:
So, since we discovered our config will not work unless Zookeeper is running, I think we might want to switch to a different config file in dist mode rather than using the same one. I would suggest adding a new setting e.g. SENTRY_DISTRIBUTED_CLICKHOUSE_TABLES and use it to switch between the regular and distributed Clickhouse config. Basically the same idea as https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L1576-L1584.